src/cloudStorageProvider.js

```diff
@@ -145,6 +156,15 @@ class CloudStorageProvider {
                 vscode.window.showErrorMessage(`Failed to create a file: ${error.message}`);
             }
         } else {
+            // remove reference line before saving
+            const config = vscode.workspace.getConfiguration('playcanvas');
+
+            if (config.get('usePlaycanvasTypes') && asset.file.filename.endsWith('.js')) {
```

Comment:
should we handle new .mjs type as well @marklundin

Reply:
Yep good point. Valid script files are now both `.mjs` and `.js`

Reply:
added mjs to conditions